src/t2iapi/context/service.proto

```diff
@@ -129,9 +129,13 @@ service ContextService {
       returns (BasicResponse);
 
   /*
-  Indicate membership in an SDC PARTICIPANT ENSEMBLE using an ensemble context that has the provided descriptor handle
-  and return a list of pm:Identification elements which identify an ensemble and thus enable the caller to distinguish
-  ensembles from each other.
+  Indicate membership in an SDC PARTICIPANT ENSEMBLE using ensemble context state(s) that have the provided descriptor
```

Comment:
it should also mentioned that associated ensemble context state(s) are meant.
e.g. 
Indicate membership in an SDC PARTICIPANT ENSEMBLE using associated ensemble context state(s) that

Reply:
adapted - done